backend/db/test-only-migrations/R__Test-Data-Creation.sql

```diff
@@ -1422,3 +1422,53 @@ values('CARIBOU', 'Caribou', 'Caribou', 3, true, null, user, now(), user, now())
 UPDATE species_code SET display_order = 4 where species_code = 'CARIBOU';
 UPDATE species_code SET display_order = 3 where species_code = 'BOBCAT';
 
+-- CE-771
+INSERT INTO public.complaint
+(complaint_identifier, detail_text, caller_name, caller_address, caller_email, caller_phone_1, caller_phone_2, caller_phone_3, location_summary_text, location_detailed_text, incident_utc_datetime, incident_reported_utc_timestmp, reported_by_other_text, create_user_id,  create_utc_timestamp, update_user_id,  update_utc_timestamp, reported_by_code, owned_by_agency_code, complaint_status_code, geo_organization_unit_code, location_geometry_point)
+VALUES('24-900001', 'Test information on a CO Contact General Incident Type', 'Kelsey', '135 fake st', NULL, '555-555-5565', NULL, NULL, 'Cariboo Dr', 'tester call description 8', '2024-07-11 21:18:23.336', '2024-07-11 21:18:23.336', 'test2', 'FLYWAY','2024-07-11 21:18:23.336', 'FLYWAY','2024-07-11 21:18:23.336', 'BYLAW', 'COS', 'OPEN', 'VICTORIA', 'SRID=4326;POINT (0 0)'::public.geometry)
```

Comment:
This won't work in the test environment as this ID (and subsequent) will be in use.   Could prefix them with 23?

Reply:
Script updated to use 23 rather than 24 as a prefix.